testkit/src/main/scala/scorex/testkit/properties/NodeViewSynchronizerTests.scala

```diff
@@ -92,17 +92,6 @@ trait NodeViewSynchronizerTests[P <: Proposition,
     vhProbe.fishForMessage(3 seconds) { case m => m == GetSyncInfo }
   }
 
-  property("NodeViewSynchronizer: CurrentSyncInfo") { ctx =>
```

Comment:
Add test for current logic instead of this one?

Reply:
the test checked behavior which was removed for being problematic. New tests will be later, they're non-trivial